core/core_modules/kpm.js

```diff
@@ -13,6 +11,41 @@ var git = require.once('../git.js'),
         lastUpdated: null,
         modules: {}
     },
+
+    getModuleList = function(cacheOverride) {
+        if (cacheOverride === true || moduleCache === null) {
+            var mods = exports.platform.modulesLoader.listModules(true);
+            for (var m in mods) {
+                var s = mods[m].folderPath.split(path.sep);
+                if (!s[s.length - 1].startsWith('kpm_')) {
+                    delete mods[m];
+                }
+            }
+            moduleCache = mods;
+        }
+        return moduleCache;
+    },
+
+    parseRuntimeModuleList = function(args, cmd, api, event) {
+        var updateMods = getModuleList();
+        if (args.length > 0) {
+            var m = {};
+            for (var i = 0; i < args.length; i++) {
+                if (!isModuleName(args[i])) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["isModuleName" was used before it was defined](https://www.codacy.com/app/theprojectshop/Kassy/file/2589031845/issues/source?bid=3282987&fileBranchId=3291254#l34)